cocli/cmd/cotsCreateCorim.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/veraison/corim/cots"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/corim/corim"
+)
+
+var (
+	cotsCorimCreateCorimFile  *string
+	cotsCorimCreateCotsFile   *string
+	cotsCorimCreateOutputFile *string
+	cotsCorimCreateCotsFiles  []string
```

Comment:
The list of files is not in use at the moment, we can get rid of it, if not using it!